client/src/components/LikeButton.tsx

```diff
@@ -1,51 +1,32 @@
 import { useState } from 'react';
 import { PiHeartFill } from 'react-icons/pi';
 import { styled } from 'styled-components';
-import client, { ClientNetworkError } from '../client';
-import { Cafe } from '../types';
 
-type LikeProps = {
-  currentLikeCount: number;
+type LikeButtonProps = {
+  likeCount: number;
 };
```

Comment:
```suggestion
type LikeButtonProps = {
  likeCount: number;
  onChange: (likeCount: number) => void;
};
```
사용자가 버튼을 클릭했을 때 부모 컴포넌트가 알 수 있도록 콜백 함수를 props로 추가하는 것이 좋을 것 같아요

Reply:
넷넷 추가 했고 임의로 Home에도 관련된 함수 넣었습니다!

Reply:
너무 좋습니다